articles/quickstart/webapp/nodejs/01-login.md

```diff
@@ -14,4 +16,239 @@ github:
 ---
 <%= include('../_includes/_getting_started', { library: 'Node.js', callback: 'http://localhost:3000/callback' }) %>
 
-<%= include('_includes/_login') %>
+## Configure Node.js to Use Auth0 
+
+### Install and load the Dependencies
+
+To follow along this guide, install the following dependencies.
+
+* [passport](http://www.passportjs.org/) - an authentication middleware for Node.js
+* [passport-auth0](https://github.com/auth0/passport-auth0) - an Auth0 authentication strategy for Passport
+* [express-session](https://www.npmjs.com/package/express-session) - an Express middleware to manage sessions
+* [connect-ensure-login](https://github.com/jaredhanson/connect-ensure-login) - a middleware to ensure the user is logged in order to access certain routes
+
+```bash
+# installation with npm
+npm install passport passport-auth0 express-session connect-ensure-login --save
+
+# installation with yarn
+yarn add passport passport-auth0 express-session connect-ensure-login
+```
+
+### Configure express-session
+
+In `app.js`, include the `express-session` module and configure it. The `secret` parameter is a secret string that will be used to sign the session ID cookie. Please use a custom value.
+
+```js
+// app.js
+
+var session = require('express-session');
+
+//session-related stuff
+var sess = {
+ secret: 'CHANGE THIS SECRET',
+ cookie: {},
+ resave: false,
+ saveUninitialized: true
+};
+
+if (app.get('env') === 'production') {
+ sess.cookie.secure = true; // serve secure cookies, requires https
+}
+
+app.use(session(sess));
+```
+
+### Configure Passport with the Auth0 Strategy and Your Auth0 Application Settings
+
+In `app.js`, include the `passport` and `passport0-auth0`modules, configure Passport to use a new instance of `Auth0Strategy` with your Auth0 application settings. Load the `passport.initialize()` and `passport.session()` middlewares to initialize Passport with persistent login sessions. 
```

Comment:
modules, *and* configure Passport

(Add the "and")

>Load the `passport.initialize()` and `passport.session()` middlewares to initialize Passport with persistent login sessions.

I don't think we are 'loading them to initialize them', maybe 'Use the ..'